src/components/mailing-list/mailing-list-full.astro

```diff
@@ -16,12 +16,6 @@
 		provide good insights to the content we have.
 	</p>
 	<div data-style="clean">
-		<ul
-			class="formkit-alert formkit-alert-error"
-			data-element="errors"
-			data-group="alert"
-		>
-		</ul>
```

Comment:
These need to stay here. I'll have a follow-up PR to fix the issue. Let's revert this change for this PR